docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Check your operating system and its version: 
+   ```sh 
+   cat /etc/os-release
    ```
 
-2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install it, run this script:
-   ```sh
-   sudo -E ./install_NEO_OCL_driver.sh
-   ```
-   > **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
-   
-   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does 
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
+
+Operation System | Driver version
+--- |-------------------------
```

Comment:
Do we have intention of updating this every time when we release something? I think this is not entirely sustainable process and perhaps we need to find some other way to tell which version we used during our validation. Now you are interrogating with installation instruction which can actually point to something else than GitHub repo.